system/Database/SQLite3/Forge.php

```diff
@@ -110,6 +110,31 @@ public function dropDatabase(string $dbName): bool
         return true;
     }
 
+    /**
+     * @param list<string>|string $columnNames
+     *
+     * @throws DatabaseException
+     */
+    public function dropColumn(string $table, $columnNames): bool
+    {
+        $columns = is_array($columnNames) ? $columnNames : array_map(trim(...), explode(',', $columnNames));
```

Comment:
This seems redundant as `Table::dropColum()` seems to already handle this.

Reply:
Yes, but this is also used in the exception message formatting.

Reply:
Ok, I get your motivation now.